include/glow/Runtime/Provisioner/Provisioner.h

```diff
@@ -45,7 +45,7 @@ class Provisioner final {
 private:
   /// Pointer to backend used for compilation. This currently gets reset per
   /// device to ensure the correct backed per device.
-  std::unique_ptr<Backend> backend_;
+  std::vector<std::unique_ptr<Backend>> backends_;
```

Comment:
Do you need a copy of the backends vector here? Or would `std::vector<std::unique_ptr<Backend>> &backends_` be enough?

Reply:
This one is not passed from HostManager, all the backends are created in Provisioner.